webpack.config.js

```diff
@@ -105,7 +105,7 @@ const config = (module.exports = {
           ]
         : []),
       {
-        test: /\.(eot|woff2?|ttf|svg|png)$/,
+        test: /\.(svg|png)$/,
```

Comment:
We're not directly loading the fonts anymore (since `fonts.module.css` has been deleted) so in theory we don't need to worry about including any of the font files in the build process